contents/docs/data-warehouse/join.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Filtering, visualizing, and joining data
+sidebar: Docs
+showTitle: true
+availability:
+  free: full
+  selfServe: full
+  enterprise: full
+---
+
+Your external data can be used alongside the rest of the PostHog.
```

Comment:
We should decide on a verb you want to use. 

- Link (used in other docs)
- Sync (used later in this doc)
- Connect (used by connectors)

Maybe link refers to initial connection and sync refers to ongoing connection, but that means we should remove "connect."

```suggestion
Once you [link a source](/docs/data-warehouse/setup), your external data can be used alongside the rest of the PostHog.
```